### ISSUE TITLE
Doubt about the problem

### ISSUE THREAD

**Issue by pauloabmiranda** [talkdesk-challenges/challenge_paulomiranda#1]
In the example of the data sent on a call event there is no information about the location so we can map the calls in the globe. 
Can i add it or is another call to get the data using customer_id and agent_id.

Thanks in advance.

**Comment by yarcub** [comment]
Hi @pauloabmiranda.

That's one part of the problem. Location is not present in the system but the event contains data that we could use to infer a location, even if not the more accurate one.

From the sample data we can have a good guess on the country it originated from.

Hope it helped you move forward. Have fun!

**Comment by pauloabmiranda** [comment]
It helped, so it doesn't have to be accurate just by country.
If the customer and the agent are in the same country what kind of visualisation will happen, just a bar? 

I will have to simulate the phone calls, the data can be random generated by me or do i have to use any provided data?

**Comment by yarcub** [comment]
> It helped, so it doesn't have to be accurate just by country.

Yeah, with the available data we can't have an accurate location. But, on some cases, it's possible to infer more than just a country. For example:
* could `+35121******` be from Faro area?

It's not a new problem so I suggest you to look how others are doing it and probably you could build on top of existent solutions. Don't get stuck on a problem for too long, get a level of accuracy you're comfortable with and move on.

> I will have to simulate the phone calls, the data can be random generated by me or do i have to use any provided data?

You can generate, but make sure it's valid data (e.g. valid phone numbers). 😉 

> If the customer and the agent are in the same country what kind of visualisation will happen, just a bar?

That's part of challenge, i'm looking forward to see the solution you end up with.  😁
